core/jazz_codeq/test/test.js

```diff
@@ -649,10 +672,10 @@ describe('utils', () => {
 					return obj.callback(null, response, body);
 				});
 
-				const expectedResult = utils.getCodeqReport(metrics, "master", "todate", "fromdate", query, config)
+				const expectedResult = utils.getCodeqReport(metrics, "master", "todate", "fromdate", query, config , serviceContext)
 				.then(result => {
 					expect(result.metrics.length).to.eq(1);
-					expect(result.metrics[0].link).to.eq('serviceurl/helpurl?metrics=security');
+					expect(result.metrics[0].link).to.eq('NaN/component_measures?id=jazz_test_test_master&metric=security');
```

Comment:
One of the unit tests is failing, can you check?

Reply:
fixed this